publish-service/index.js

```diff
@@ -15,7 +16,11 @@ function doPublish(cb) {
 
 export function publish(event, context, cb) {
   try {
-    doPublish(cb);
+    if (event && event.body && event.body.secret === secretAuth) {
```

Comment:
I was conscious of 'cannot get body of undefined' errors - but I think it's highly unlikely to get undefined. You can get an empty body but probably not an actual undefined.
Will remove it.